src/annotator/components/toolbar.js

```diff
@@ -0,0 +1,135 @@
+import classnames from 'classnames';
+import propTypes from 'prop-types';
+import { createElement } from 'preact';
+
+function ToolbarButton({
+  buttonRef,
+  extraClasses,
+  label,
+  icon,
+  onClick,
+  selected,
+}) {
+  const handleClick = event => {
+    event.stopPropagation();
+    onClick();
+  };
+
+  return (
+    <button
+      // This currently uses the icon font. This needs to be converted to
+      // `SvgIcon`.
+      className={classnames('annotator-frame-button', extraClasses, icon)}
+      aria-label={label}
+      aria-pressed={selected}
+      onClick={handleClick}
+      ref={buttonRef}
+      title={label}
+    />
+  );
+}
+
+ToolbarButton.propTypes = {
+  buttonRef: propTypes.any,
+  extraClasses: propTypes.string,
+  label: propTypes.string.isRequired,
+  icon: propTypes.string.isRequired,
+  onClick: propTypes.func.isRequired,
+  selected: propTypes.bool,
+};
+
+/**
+ * Controls on the edge of the sidebar for opening/closing the sidebar,
+ * controlling highlight visibility and creating new page notes.
+ */
+export default function Toolbar({
+  closeSidebar,
+  createAnnotation,
+  isSidebarOpen,
+  newAnnotationType,
+  showHighlights,
+  toggleHighlights,
+  toggleSidebar,
+  toggleSidebarRef,
+  useMinimalControls = false,
+}) {
+  return (
+    <div>
```

Comment:
This `div` doesn't need a `role` or anything, does it? We've kind of got a set of controls here, so thought I'd ask...

Reply:
We could make this an ARIA toolbar. If we do that though, we need to implement the correct keyboard interaction behavior at the same time.